password_renewal_bp.py

```diff
@@ -1,37 +1,42 @@
 from flask import Blueprint, render_template, request, redirect, flash, abort, url_for, g
 from flask_login import current_user, logout_user
 import os
-import db_funkce
+import db_functions
 from cryptography.fernet import Fernet, InvalidToken
 import json
-import posta_funkce
+import email_functions
+
+blueprint = Blueprint('password_renewal_bp', __name__)
+
 
-blueprint = Blueprint('obnova_hesla_bp', __name__)
 @blueprint.route('/obnovahesla')
-def show_obnova():
+def show_renewal():
 	return render_template('obnova_hesla.html')
 
 
 f = Fernet(os.environ["PASSWORD_RESET_KEY"].encode("ascii"))
```

Comment:
Since we are renaming variables, please rename `f` to `pw_refresh_token_generator`

Reply:
(or something among those lines)